draft-wang-ppm-dap-taskprov.md

```diff
@@ -570,13 +570,26 @@ Author to be under control of the adversary. It is therefore incumbent on
 protocol participants to verify the privacy parameters of a task before opting
 in.
 
-Another risk is that a malicious coalition of Clients might attempt to pollute
-an Aggregator's long-term storage by uploading reports for many (thousands or
-perhaps millions) of distinct tasks. While this does not directly impact tasks
-used by honest Clients, it does present a Denial-of-Service risk for the
-Aggregators themselves. This can be mitigated by limiting the rate at which new
-tasks or configured. In addition, deployments SHOULD arrange for the Author to
-digitally sign the task configuration so that Clients cannot forge task creation.
+Another risk introduced by the addition of the Author is the Author could
+configure tasks uniquely for a Client to fingerprint the Client. If the Author
+colludes with the Leader, it can learn some metadata information about the
+Client, e.g., the Client IP and user agent string. This can be mitigated by
+anonymizing the Client, such as by using Oblivious HTTP
+{{!I-D.draft-ietf-ohai-ohttp-08}} in the DAP upload route. We also note that,
```

Comment:
It's not clear to me that OHTTP mitigates this issue. In fact, I think the converse is true: fingerprinting the client effectively removes any protection that the client gets from OHTTP. Concretely, the Author configures the client with `TaskConfig<task_info = "this client has IP address 192.168.0.1">`; when the Leader gets the Client's report (via the OHTTP proxy), the Leader immediately knows the IP address of the client that produced the report.

Reply:
You mean only targeting that client in a particular IP address. Let's avoid talking about OHTTP mitigation in this scenario then. Instead we will just say, if the author can deanonymize the client, the author can learn the metadata information at best, if the client carefully checks for privacy parameter.